kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/group/GroupMetadata.java

```diff
@@ -598,6 +605,27 @@ public boolean hasPendingOffsetCommitsFromProducer(long producerId) {
         ));
     }
 
+    public Set<TopicPartition> collectPartitionsWithTopics(Set<String> topics) {
```

Comment:
Maybe it's not a big issue. Since the collection only calls `addAll` method, it's better to use `List` rather than `Set`.